app/controllers/admin/logs_controller.rb

```diff
@@ -9,13 +9,21 @@ def show
       @result = []
       return redirect_to admin_logs_index_path, notice: t(:no_logs) unless @files.include?(@path)
 
-      File.open(@path) do |file|
-        file.read.split("\n").each do |line|
-          @result.push line
+      require 'io'
+      open(@path) do |io|
```

Comment:
__[Brakeman]__

Parameter value used in file name


<!-- Comment by [SideCI](https://sideci.com) -->